README.md

```diff
@@ -1102,6 +1107,49 @@ INFO     Linting module: star/align
 ╰──────────────────────╯
 ```
 
+### Bump bioconda and container versions of modules in
+
+If you are contributing to the `nf-core/modules` repository and want to bump bioconda and container versions of certain modules, you can use the `nf-core modules bump-versions` helper tool. This will bump the bioconda version of a single or all modules to the latest version and also fetch the correct Docker and Singularity container tags.
+
+```console
+$ nf-core modules bump-versions -d modules 
+
+                                          ,--./,-.
+          ___     __   __   __   ___     /,-._.--~\
+    |\ | |__  __ /  ` /  \ |__) |__         }  {
+    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
+                                          `._,._,'
+
+    nf-core/tools version 1.15.dev0
```

Comment:
```suggestion
    nf-core/tools version 2.0.dev0
```
Not sure if this is the correct thing to do ...

Reply:
Yup that works, could actually even remove the dev tag here